source/admin_manual/doveadm_http_api.rst

```diff
@@ -3329,31 +3329,39 @@ parameters::
-+------------+---------+---------------------------------------+---------------------------------+
-| key        | String  | metadata key to get                   |                                 |
-+------------+---------+---------------------------------------+---------------------------------+
-| keyPrefix  | String  | search prefix for keys                |                                 |
-+------------+---------+---------------------------------------+---------------------------------+
-| mailbox    | String  | mailbox to fetch metadata from        |                                 |
-+------------+---------+---------------------------------------+---------------------------------+
-
++----------------+---------+---------------------------------------+---------------------------------+
+| Parameter      | Type    | Description                           | example                         |
++================+=========+=======================================+=================================+
+| socketPath     | String  | Path to doveadm socket                | /var/run/dovecot/doveadm-server |
++----------------+---------+---------------------------------------+---------------------------------+
+| allUsers       | Boolean | apply operation to all users          |                                 |
++----------------+---------+---------------------------------------+---------------------------------+
+| user           | String  | uid to apply metadata get             |                                 |
++----------------+---------+---------------------------------------+---------------------------------+
+| userFile       | String  | optionally fetch usernames from file. |                                 |
+|                |         | One username per line                 |                                 |
++----------------+---------+---------------------------------------+---------------------------------+
+| key            | String  | metadata key to get                   |                                 |
++----------------+---------+---------------------------------------+---------------------------------+
+| keyPrefix      | String  | search prefix for keys                |                                 |
++----------------+---------+---------------------------------------+---------------------------------+
+| mailbox        | String  | mailbox to fetch metadata from        |                                 |
++----------------+---------+---------------------------------------+---------------------------------+
+| prepend-prefix | Boolean | Prepend metadata type prefix          |                                 |
+|                |         | ("/shared" or "/private") to name     |                                 |
++----------------+---------+---------------------------------------+---------------------------------+
+
+   .. versionadded:: v2.3.14 New parameter "prepend-prefix" added.
```

Comment:
You can add "versionadded" inside the table, like:

```
|                |         | ("/shared" or "/private") to name     |                                 |
|                |         |                                       |                                 |
|                |         | .. versionadded:: ...                 |                                 |
```

```

Reply:
aha, didn't know that. Changed.